qiskit_experiments/library/characterization/analysis/readout_angle_analysis.py

```diff
@@ -14,31 +14,92 @@
 """
 
 import numpy as np
+from typing import List, Optional
 
-from qiskit_experiments.framework import BaseAnalysis, AnalysisResultData
-
+from qiskit_experiments.framework import BaseAnalysis, AnalysisResultData, Options
+from qiskit_experiments.framework.matplotlib import get_non_gui_ax
```

Comment:
I'm not familiar with this one, so someone who's familiar should check it. If other experiments work in other ways then it's suspicious.

Reply:
1. I ran the experiment on a device and verified the results.
2. How do you load an experiment from the web?

Reply:
Regarding `get_non_gui_ax`, I found this was used by `curve_analysis`. Perhaps @nkanazawa1989 can answer whether this is correct usage here.

Reply:
Regarding your comment @yaelbh about other moving other customAnalysis functions, I will do that in a separate PR.

Reply:
@nkanazawa1989 , can you please check if the usage of `get_non_gui_ax` is correct here?

Reply:
Did you get an answer from @nkanazawa1989 ?